pkg/moov/representative_api.go

```diff
@@ -0,0 +1,75 @@
+package moov
+
+import (
+	"context"
+	"errors"
+	"net/http"
+)
+
+// CreateRepresentative creates a new representative for a given account
+func (c Client) CreateRepresentative(ctx context.Context, accountID string, representative CreateRepresentative) (*Representative, error) {
+	resp, err := c.CallHttp(ctx,
+		Endpoint(http.MethodPost, pathRepresentatives, accountID),
+		AcceptJson(),
+		WaitFor("connection"),
```

Comment:
I don't think this endpoint handles an x-wait-for

Reply:
ah good catch - thats a copypasta mistake on my part. Updated.